qiskit/ignis/experiments/quantum_volume/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Quantum Volume Experiment
+"""
+
+from .qv_experiment import (QuantumVolumeExperiment,
+                            QuantumVolumeGenerator,
+                            QuantumVolumeAnalysis)
```

Comment:
QV experiments contain several plots - where should they be added?

Reply:
for now I added them in the result object